From d7bce9c2e2aab2e62f2d4eba7385fa0d8c889c66 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jos=C3=A9=20Ignacio=20Alamos=20Aste?= <jialamos@uc.cl>
Date: Mon, 3 Oct 2016 16:22:13 -0300
Subject: [PATCH] fixed mbedtls config

---
 third_party/mbedtls/mbedtls-config.h | 4 +-
 1 file changed, 1 insertions(+), 1 deletions(-)

diff --git a/third_party/mbedtls/mbedtls-config.h b/third_party/mbedtls/mbedtls-config.h
index 4f80774..762ee24 100644
--- a/third_party/mbedtls/mbedtls-config.h
+++ b/third_party/mbedtls/mbedtls-config.h
@@ -2562,6 +2562,6 @@ __inline int windows_kernel_snprintf(char * s, size_t n, const char * format, ..
 #include MBEDTLS_USER_CONFIG_FILE
 #endif
 
-#include "check_config.h"
+#include "mbedtls/check_config.h"
 
 #endif /* MBEDTLS_CONFIG_H */

